.github/scripts/windows/build.bat

```diff
@@ -0,0 +1,28 @@
+@echo on
```

Comment:
I don't think this is a good idea; while it might help with debugging, it's noisy as hell.
```suggestion
@echo off
```
Same for the other files.

Reply:
this is equivalent to `set -x` used in linux actions

it helps understanding of what is beiiing executed, I would love to see the /wo & /w opcache testing separated into 2 GH Actions steps, but would it be ok to merge this more 1:1 AppVeyor code reuse first?

> There is something wrong regarding the number of passing test; while the new AppVeyor CI and the GH actions CI pass roughly 11,000 tests, the current AppVeyor CI passes more than 14,000. Did we forget to build some extensions? And generally, I wouldn't have expected that the AppVeyor CI has changed at all; it better should stay as is for a while, and finally be retired when we're comfortable relying on GH action CI for Windows only.

AppVeyor CI is untouched by this PR, only the scripts are deduplicated & simplified

about the number of tests run on Github Actions - services defined in https://github.com/php/php-src/blob/f22608913c702366d4656f98be26633cd086a4db/.appveyor.yml#L48-L50 are not ported, can this PR be merged first? I would like to define the services once per all GH tests, eg. for linux too, an I think it is better to do in another PR, is that ok with you?

Reply:
Currently, we having 14,000 test passing. After merge, we only have 11,000 tests passing. This is a no-go. Why not leave AppVeyor CI alone, and just duplicate the files needed for GH actions?

Reply:
at least because it can be used it in forks for testing easily...

ok - I will add the required services in this PR

Reply:
unfortunately native GH services are not supported with Windows images:

![image](https://user-images.githubusercontent.com/2228672/165122234-0dc2ca5c-b00c-4920-8460-5f7d8007990d.png)

I will reuse some code from https://github.com/flavioheleno/php-src/commit/0ee6bc5d58ee17e5404fe1252a3ce5c9891e1d75